src/glfw3.jl

```diff
@@ -298,6 +385,39 @@ immutable GLFWError <: Exception
 end
 Base.showerror(io::IO, e::GLFWError) = print(io, "GLFWError ($(e.code)): ", e.description)
 
+#Came from GLWindow.jl/types.jl
+struct MonitorProperties 
```

Comment:
Change `struct` to `immutable` for Julia 0.5 compatibility.

Reply:
This request was made irrelevant by #121. However, I couldn't get GitHub to resolve merges through it's UI, so I did it manually and push it to #122.